srcpkgs/py3status/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'py3status'
 pkgname=py3status
-version=3.28
+version=3.29
 revision=1
 archs=noarch
```

Comment:
`noarch` is deprecated, please remove this line.

Reply:
done